src/test/java/com/ionosenterprise/rest/test/SnapshotTest.java

```diff
@@ -84,7 +80,7 @@ public static void t1_createSnapshot() throws RestClientException, IOException,
 
         Snapshot.Properties properties = SnapshotResource.getSnapshot().getProperties();
         Snapshot snapshot = ionosEnterpriseApi.getSnapshot().createSnapshot(dataCenterId, volumeId,
-                properties.getName(), properties.getDescription());
+                properties.getName(), properties.getDescription(), properties.getLicenceType().name());
         assertNotNull(snapshot);
         assertEquals(snapshot.getProperties().getName(), properties.getName());
         assertEquals(snapshot.getProperties().getDescription(), properties.getDescription());

```

Comment:
i think we also need an assert here for the licence type , correct me if i am wrong

Reply:
You are right. Fixed.